console/src/components/Services/Data/utils.js

```diff
@@ -790,13 +790,21 @@ WHERE
 export const isColTypeString = colType =>
   ['text', 'varchar', 'char', 'bpchar', 'name'].includes(colType);
 
-export const cascadeUpQueries = (upQueries = []) =>
+const cascadePGSqlQuery = sql => {
```

Comment:
A lot is going on here. Can you put curly brackets for readability?

What about:
```js
const cascadePGSqlQuery = sql => {
  if (sql[sql.length - 1] === ';') {
    return sql.substr(0, sql.length - 1) + 'CASCADE';
  }
  // SQL might have  a " at the end
  return sql.substr(0, sql.length - 2) + ' CASCADE';
};

```

Reply:
Done